client/src/components/Weather/HourCard/HourCard.module.scss

```diff
@@ -0,0 +1,6 @@
+/*.hourCard{
```

Comment:
Leaving commented code in the codebase is confusing. I notice that you are still using the hourCard class styling on line 78 of the HourCard/index.js file. Were you meant to uncomment this code?

Reply:
fix now